c7n/resources/rds.py

```diff
@@ -475,11 +475,17 @@ def process_tag_removal(self, client, resource, candidates):
 
 
 def _eligible_start_stop(db, state="available"):
-
+    # See conditions noted here
+    # https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_StopInstance.html
+    # Note that this doesn't really specify what happens for all the nosql engines
+    # that are available as rds engines.
     if db.get('DBInstanceStatus') != state:
         return False
 
-    if db.get('MultiAZ'):
+    if db.get('MultiAZ') and db['Engine'].startswith('sqlserver-'):
+        return False
+
+    if db['Engine'] in ('docdb', 'neptune'):
```

Comment:
imo would be better as a whitelist as that's what the docs provide, e.g. 
```
db['Engine'] not in ('postgresql', 'mariadb'...)
```

Reply:
okay.. fwiw the full set of engines is {'postgres', 'sqlserver-ee', 'oracle-se2', 'mariadb', 'docdb', 'oracle-ee', 'sqlserver-ex', 'sqlserver-se', 'oracle-se', 'aurora-mysql', 'mysql', 'oracle-se1', 'aurora-postgresql', 'sqlserver-web', 'aurora', 'neptune'}

Reply:
ie this applies to every single rdbms rds engine